modules/social_features/social_event/modules/social_event_invite/social_event_invite.install

```diff
@@ -18,12 +18,7 @@ function social_event_invite_install() {
   /** @var \Drupal\group\Entity\GroupType $group_type */
   foreach (GroupType::loadMultiple() as $group_type) {
     // We do not want to enable the invite feature for closed or secret groups.
```

Comment:
This loop can probably be simplified since there's no more logic in it. We may want to flag the "Event invite" for a follow-up to see how this is added to group (turning it into a group plugin rather than custom settings).